lib/convection/control/stack.rb

```diff
@@ -359,6 +402,14 @@ def cf_tags
           }
         end
       end
+
+      def hash_with_array_as_default
+        Hash.new { |hash, key| hash[key] = [] }
+      end
+
+      def reversed_hash(hash)
```

Comment:
`Stack#reversed_hash` and `Stack#hash_with_array_as_default` are no longer used.